src/commands/dora/api.ts

```diff
@@ -1,11 +1,11 @@
 import type {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios'
 
+import {getDatadogSite} from '../../helpers/api'
 import {getRequestBuilder} from '../../helpers/utils'
 
 import {DeploymentEvent} from './interfaces'
 
-export const datadogSite = process.env.DD_SITE || 'datadoghq.com'
-export const apiUrl = `https://api.${datadogSite}`
+export const apiUrl = `https://api.${getDatadogSite}`
```

Comment:
I'm impressed this hasn't made any unit test fail 😅 
```suggestion
export const apiUrl = `https://api.${getDatadogSite()}`
```

Reply:
We could add a test in the `describe('getApiHelper` of each command ensuring that the endpoint is the expected one

Reply:
yeah absolutely!